ya-runtime-erigon/src/main.rs

```diff
@@ -84,9 +103,13 @@ impl Runtime for ErigonRuntime {
     }
 
     fn start<'a>(&mut self, ctx: &mut Context<Self>) -> OutputResponse<'a> {
-        //TODO: Receive chain_id from start parameters when Yapapi ready
-        // see: https://github.com/golemfactory/yapapi/issues/390
-        let chain_id = String::from("goerli");
+        let default_chain_id = "goerli".to_string();
+        let chain_id = if let ya_runtime_sdk::cli::Command::Start { args } = &ctx.cli.command {
+            parse_json(args.into_iter().next(), &default_chain_id)
```

Comment:
Is really `args.into_iter().next()` the proper way to get command line params in `ya_runtime_sdk`? Looks hacky. Summon @mfranciszkiewicz

Reply:
@Wiezzel `args: Vec<String>`

Reply:
It will change in [APPS-152](https://golemproject.atlassian.net/browse/APPS-152)